bin/www.ts

```diff
@@ -3,9 +3,10 @@
 /**
  * Module dependencies.
  */
-const app = require('../app')
-const debug = require('debug')('fake-log:server')
-const http = require('http')
+import app from '../app'
+import debug from 'debug'
+// ('fake-log:server')
```

Comment:
```suggestion
```

Reply:
I would like you to confirm this one here
original file had 
`const debug = require('debug')('fake-log:server')`

After converting to import-from style I couldn't figure out where to put this call since the project was running fine like before changing anything

Reply:
I think your changes are fine. Previous version was maintained by somebody else than me, so I also do not really understand everything